docs/solidity-by-example.rst

```diff
@@ -1079,16 +1012,13 @@ The full contract
     }
 
 
-Note: The function ``splitSignature`` is very simple and does not use all security checks.
-A real implementation should use a more rigorously tested library, such as
-openzepplin's `version <https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/ECRecovery.sol>`_ of this code.
-
-  
+.. note::
+  The function ``splitSignature`` is very simple and does not use all security checks. A real implementation should use a more rigorously tested library, such as openzepplin's `version <https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/ECRecovery.sol>`_ of this code.
```

Comment:
Why change it from manually broken lines?

There are two problems with this:
- diffs are harder to see
- it is painful in some editors

Reply:
@axic The whole line break issue is a big conversation between writers and coders, but in all honestly it's mostly because I haven't worked on a code base for a while that hard enforced line breaks in docs and had just got out of the habit, fixed.